### PR TITLE
Update Readme to describe how to generate an ERD

### PR DESCRIPTION
**Pull request by lamorrison**
Adds a gem (rails-erd) which creates diagrams of the database. Very informative!

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -74,6 +74,8 @@ group :development do
 
   # opens emails for us
   gem 'letter_opener'
+
+  gem 'rails-erd'
```
Comment (Br3nda) [inline Gemfile]: How about a comment to explain what the gem does for us. Helps when one day we are attempting to clean out gems we don't use anymore

File: README.md [hunk 2]
```diff
@@ -80,6 +80,13 @@ Rubocop
 bundle exec rubocop
 ```
 
+## Database structure
+
+Generate a PDF diagram of the database structure (`rails-erd` gem) in the root folder of the application:
+```
+rake erd
```
Comment (Br3nda) [inline README.md]: doesn't this need a bundle command ?

File: Gemfile (hunk 1 above)
Comment (dlouise64) [reply Gemfile]: @lamorrison I have to install GraphViz, can we mention this here too?

**Comment by Br3nda** [comment]
The ci is hitting OF and not getting the result it expects. Can someone look into that?

File: Gemfile [hunk 3]
```diff
@@ -74,6 +74,10 @@ group :development do
 
   # opens emails for us
   gem 'letter_opener'
+
+  # genereates an Entity Relationship Diagram (/erd.pdf) of the database
```
Comment (mermop) [inline Gemfile]:
```suggestion
  # generates an Entity Relationship Diagram (/erd.pdf) of the database
```

**Pull request merged**